pkg/deploy/defaults.go

```diff
@@ -30,21 +30,23 @@ const (
 	DefaultPvcClaimSize                 = "1Gi"
 	DefaultIngressStrategy              = "multi-host"
 	DefaultIngressClass                 = "nginx"
-	DefaultPluginRegistryImage          = "quay.io/eclipse/che-plugin-registry:7.0.0"
+	defaultPluginRegistryImage          = "registry.redhat.io/codeready-workspaces/che-plugin-registry:2.0"
+	defaultPluginRegistryUpstreamImage  = "quay.io/eclipse/che-plugin-registry:7.0.0"
 	DefaultPluginRegistryMemoryLimit    = "256Mi"
 	DefaultPluginRegistryMemoryRequest  = "16Mi"
-	DefaultDevfileRegistryImage         = "quay.io/eclipse/che-devfile-registry:7.0.0"
+	defaultDevfileRegistryImage         = "registry.redhat.io/codeready-workspaces/che-devfile-registry:2.0"
+	defaultDevfileRegistryUpstreamImage = "quay.io/eclipse/che-devfile-registry:7.0.0"
 	DefaultDevfileRegistryMemoryLimit   = "256Mi"
 	DefaultDevfileRegistryMemoryRequest = "16Mi"
 	DefaultKeycloakAdminUserName        = "admin"
 	DefaultCheLogLevel                  = "INFO"
 	DefaultCheDebug                     = "false"
-	DefaultPvcJobsImage                 = "registry.redhat.io/ubi8-minimal:8.0-159"
-	DefaultPvcJobsUpstreamImage         = "registry.access.redhat.com/ubi8-minimal:8.0-159"
-	DefaultPostgresImage                = "registry.redhat.io/rhscl/postgresql-96-rhel7:1-46"
-	DefaultPostgresUpstreamImage        = "centos/postgresql-96-centos7:9.6"
-	DefaultKeycloakImage                = "registry.redhat.io/redhat-sso-7/sso73-openshift:1.0-13"
-	DefaultKeycloakUpstreamImage        = "eclipse/che-keycloak:7.0.0"
+	defaultPvcJobsImage                 = "registry.redhat.io/ubi8-minimal:8.0-159"
```

Comment:
the subtlety of upper and lower case vars here might be lost on people... is there no better way to define the distinction?

Reply:
Well, lowercase is because these are not shared.
uppercase means that they are shared outside of the package.